### PR TITLE
Add type conversion into PrebidServer to handle inconsistent types.

### PR DESCRIPTION
**Pull request by mkendall07**
## Type of change
- [x] Feature

## Description of change
Handles types for different bidders properly. 

## Other information
@prebid/core for review.

### CODE REVIEW

**Comment by mkendall07** [comment]
Types should match the types found in : https://github.com/prebid/prebid-server/tree/master/adapters

File: src/adapters/prebidServer.js [hunk 1]
```diff
@@ -21,9 +45,39 @@ function PrebidServer() {
...
+    adUnits.forEach(adUnit => {
+      adUnit.bids.forEach(bid => {
+        const types = paramTypes[bid.bidder] || [];
+        Object.keys(types).forEach(key => {
+          if(bid.params[key] && typeof bid.params[key] !== types[key]) {
+            //mismatch type. Try to fix
+            utils.logMessage(`Mismatched type for Prebid Server : ${bid.bidder} : ${key}. Required Type:${types[key]}`);
+            bid.params[key] = tryConvertType(types[key], bid.params[key]);
+            //don't send invalid values
+            if(!bid.params[key]) {
```
Comment (dbemiller) [inline src/adapters/prebidServer.js]: Wouldn't this cause problems if 0 or empty strings were a valid value for some param?

Comment (mkendall07) [inline src/adapters/prebidServer.js]: Empty string shouldn't be valid but 0 might. I'll update

File: src/adapters/prebidServer.js [hunk 2]
```diff
@@ -21,9 +45,39 @@ function PrebidServer() {
...
+            }
+          }
+        });
+      });
+    });
+  }
+
+  function tryConvertType(typeToConvert, value) {
+    if(typeToConvert === 'string') {
+      return Number.toString(value);
```
Comment (mkendall07) [inline src/adapters/prebidServer.js]: No one caught this huh :)

Comment (dbemiller) [inline src/adapters/prebidServer.js]: I was going to say "weren't we requiring 80% unit test coverage on all new code?" But... dunno if that counts

Comment (mkendall07) [inline src/adapters/prebidServer.js]: It does apply. I'm writing unit tests now.

**Comment by mkendall07** [comment]
Ready for review again.

**Pull request merged**